javascript/main.js

```diff
@@ -0,0 +1,30 @@
+
+var button = document.getElementById("button");
+button.addEventListener("click", makeTree);
+document.addEventListener("keydown", enter);
+
+
+function enter(e) {
+  if (13 == e.keyCode) {
+     makeTree()
```

Comment:
Make sure you pass the event so that the make tree function has access to it.

`makeTree(e)`